sdf-feature.cddl

```diff
@@ -37,23 +34,34 @@ commonqualities = (
  ? sdfRequired: pointer-list    ; applies to qualities of properties, of data
 )
 
+arraydefinitionqualities = (
+ ? ("minItems" .feature "1.2") => number
```

Comment:
More likely integer...

Reply:
I changed the data type to `uint` now. Does this work or are there potential problems when converting the schema to JSO?

Reply:
Let's find out when we get to this (next week)...

Reply:
OK, just checked:

This becomes

``` 
    "uint": {
      "type": "number",
      "minimum": 0
    },
``` 

and is then referenced 

```
        "minItems": {
          "$ref": "#/definitions/uint"
        },
        "maxItems": {
          "$ref": "#/definitions/uint"
        },
```

Reply:
Yes, of course...